lib/i18n_txt_backend.rb

```diff
@@ -0,0 +1,63 @@
+require 'i18n'
+require 'json'
+
+# Custom i18n backend that parse our "flat_yml" files into the nested
+# hash structure that i18n works with
+class I18nTxtBackend < I18n::Backend::Simple
+  # @param filename [String] filename, assumed to have the locale slug in the filename such as "en.txt"
+  # @return [Array(Hash, Boolean)] tuple of a hash and keys_symbolized
+  def load_txt(filename)
+    locale = File.basename(filename, '.txt')
```

Comment:
The locale is guessed from the filename because it lets us drop the locale from the key in the file. Without this, the files would be like:

```
en.foo.bar.baz: "test"
```
and that would mean keys would be harder to copy-paste across files.
I